### DISCUSSION TITLE
Suggestions about UserDataStorage.CopyTo() and UserDataStorage.Clear()

### DISCUSSION THREAD

**Discussion by ReneeGA2020** [dotnet/dotNext#245] [Ideas]
Suggestion 1:
It looks like UserDataStorage.CopyTo only works when the orgin UserDataStorage has anything stored, otherwise the target object's storage will not be changed. This probably worth to be mentioned in the document, or change its behavior to clear up all the UserData in the target object.

Alternately, provide a Method to check if the storage has any UserData.

Suggestion 2:
I think UserDataStorage.Clear() would be useful.

**Comment by sakno** [comment]
Moved to issue #248.
